hera_qm/tests/test_utils.py

```diff
@@ -404,3 +404,21 @@ def test_dynamic_slice():
     nt.assert_equal(b.shape, (1, 5))
 
     nt.assert_raises(ValueError, utils.dynamic_slice, 'foo', slice(0, None))
+
+def test_process_ex_ants():
```

Comment:
This one mega test looks like it is testing the different returns and failure routes of process_ex_ants. It would be a little more helpful from a debugging standpoint if each `assert`-like statement was its own test. Then we can pinpoint issues in futures updates if there are any. I know there are a *LOT* of tests like this in the current version of `hera_qm` and a lot of it is legacy code but if we get better about writing new tests and code better there will be less technical debt later. My two cents.